dmutils/filters.py

```diff
@@ -13,3 +15,26 @@ def smartjoin(input):
         return '{}'.format(list_to_join[0])
     else:
         return ''
+
+
+def format_links(text):
+    pattern = re.compile(r"""((?:http[s]?://|www)(?:[a-zA-Z]|[0-9]|[$-_@.&#+]|[!*\(\),]|
```

Comment:
This huge regex makes me nervous.  It might be worth adding a comment to explain the intent of each part of it, or maybe naming it something more descriptive than `pattern` to make it clear what it is supposed to match.